lib/src/geo/latlng_bounds.dart

```diff
@@ -61,21 +73,21 @@ class LatLngBounds {
     required this.south,
     required this.east,
     required this.west,
-  })  : assert(
-            north <= 90, "The north latitude can't be bigger than 90: $north"),
-        assert(north >= -90,
+  })  : assert(north <= maxLatitude,
+            "The north latitude can't be bigger than 90: $north"),
```

Comment:
```suggestion
            "The north latitude can't be bigger than $maxLatitude: $north"),
```

Reply:
Thanks, you're absolutely right, the const fields should be used here too. 👍